src/HelixProject.js

```diff
@@ -208,19 +208,26 @@ class HelixProject {
   selectStrain(request) {
     // todo: use strain conditions, once implemented. for now, just use request.headers.host
     const host = request && request.headers ? request.headers.host : '';
-    const strains = this.config.strains.getByFilter(s => s.urls.find((url) => {
-      try {
-        const uri = new URL(url);
-        return uri.host === host;
-      } catch (e) {
-        // ignore
-        return false;
+    const reqPath = `${request && request.path && request.path.replace(/\/+$/, '') ? request.path : ''}/`;
+    let score = 0;
+    let best = this.config.strains.get('default');
+    this.config.strains.forEach((strain) => {
+      if (strain.urls.length > 0) {
+        const uri = new URL(strain.urls[0]);
+        if (uri.host !== host) {
+          return;
+        }
+        const uriPath = `${uri.pathname.replace(/\/+$/, '')}/`;
+        if (reqPath.indexOf(uriPath) === 0) {
+          const s = uriPath.length;
+          if (s > score) {
```

Comment:
In VCL, we simply take the first match, there is no extra score for specificity. In fact, I wouldn't even know if we can replicate this easily, so I'd also use the simpler "first to match wins" algorithm in the simulator.

Reply:
ok. but we need that a non-root path wins over a root-path. i.e. `www.project-helix.io/api/docs` wins over `www.project-helix.io/`

Reply:
solution is that the first match wins.